src/main/resources/db/migration/V1__Initial.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE products
 (
   id serial not null,
   product_name varchar(50) not null,
-  description varchar(50),
```

Comment:
you should never update already existing sql when u use flyway / liquibase and it was already imported